makefile

```diff
@@ -188,8 +188,15 @@ endif
 # single-prec codes separate, and not all have one
 EXAMPLES = $(basename $(wildcard examples/*.*))
 examples: $(EXAMPLES)
+ifneq ($(MINGW),ON)
```

Comment:
Do we all agree that a MINGW make var flag is the right way to communicate to the makefile that we're in Windows? This is simply set in ``make.inc.windows_mingw``, so we could use something else (what if we're in windows but not WSL or MinGW ?)  I am not a windows user.

Reply:
I wonder whether it would not be better to use the makefile to check if we are running on Windows?

Reply:
FWIW all other platforms are adjusted using `make.inc.*`. Unless there's a specific reason for why MinGW needs some special treatment, it would make more sense to do the same here.